client/disputes/evidence/index.js

```diff
@@ -407,14 +407,39 @@ export default ( { query } ) => {
 		updateDispute( {
 			metadata: { [ key ]: '' },
 			uploadingErrors: { [ key ]: '' },
+			fileSize: { [ key ]: 0 },
 		} );
 	};
 
+	const fileSizeExceeded = ( latestFileSize ) => {
+		const fileSizes = dispute.fileSize
+			? Object.values( dispute.fileSize )
+			: [];
+		const totalFileSize =
+			fileSizes.reduce( ( acc, fileSize ) => acc + fileSize, 0 ) +
+			latestFileSize;
+		if ( 4500000 < totalFileSize ) {
```

Comment:
Let's avoid using [magic numbers](https://www.pluralsight.com/tech-blog/avoiding-magic-numbers/) here and move 4500000 into a constant/variable with a speaking name, e.g. `disputeUploadSizeLimitInBytes`